en/02_Developer_Guides/02_Controllers/02_Routing.md

```diff
@@ -303,6 +303,97 @@ class BreadAPIController extends Controller
+
+### Example of anested request handler being returned in an action method
+
+Using the code below, navigating to the URL `/one/two/hello` will return a response with a body of "hello"
+
+```yml
+// app/_config/routes.yml
+
+SilverStripe\Control\Director:
+  rules:
+    'one': 'RequestHandlerOne'
+```
+
+```php
+// app/src/RequestHandlerOne.php
+
+use SilverStripe\Control\Controller;
+use SilverStripe\Control\HTTPRequest;
+
+class RequestHandlerOne extends Controller
+{
+    private static $allowed_actions = [
+        'two',
+    ];
+
+    public function two(HTTPRequest $request)
+    {
+        return RequestHandlerTwo::create();
+    }
+}
+```
+
+```php
+// app/src/RequestHandlerTwo.php
+
+use SilverStripe\Control\Controller;
+use SilverStripe\Control\HTTPRequest;
+use SilverStripe\Control\HTTPResponse;
+
+class RequestHandlerTwo extends Controller
+{
+    private static $allowed_actions = [
+        'hello',
+    ];
+
+    public function hello(HTTPRequest $request)
+    {
+        return HTTPResponse::create()->setBody('hello');
+    }
+}
+```
+
+### How `RequestHandler` and `Form` work together
+
+`Form` does not extend `RequestHandler`, instead it implements the [`HasRequestHandler`](api:SilverStripe\Control\HasRequestHandler) interface which defines a method `getRequestHandler()`. [`Form::getRequestHandler()`](api:SilverStripe\Forms\Form::getRequestHandler()) returns a `FormRequestHandler` which is a subclass of `RequestHandler`.
+
+Request handler subclasses and implementors of `HasRequestHandler` are treated the same because they will both end up calling `handleRequest()` on the subclass of the appropriate request handler subclass.
+
+The `FormRequestHandler.url_handlers` configuration property includes an entry `'field/$FieldName!' => 'handleField'` which allows it to handle requests to form fields on the form. [`FormRequestHandler::handleField()`](api:SilverStripe\Forms\FormRequestHandler::handleField()) will find the form field matching `$FieldName` and return it. Control is then passed to the returned form field.
+
+`FormField` extends `RequestHandler`, which means that form fields are able to handle HTTP requests and they have their own `$allowed_actions` configuration property. This allows form fields to define their own AJAX endpoints without having to rely on separately routed `RequestHandler` implementations.
+
+### Example of an AJAX form field that uses nested request handlers's
+
+The AJAX request performed by the "Viewer groups" dropdown in asset admin has an endpoint of `/admin/assets/fileEditForm/{FileID}/field/ViewerGroups/tree?format=json`
+
+That URL ends up passing the request through a series of nested request handlers, which is detailed in the step below. Unless otherwise stated, the `handleRequest()` method is called on the class that has control. Control starts with [`Director`](api:SilverStripe\Control\Director).
```

Comment:
```suggestion
That URL ends up passing the request through a series of nested request handlers, which is detailed in the steps below. Unless otherwise stated, the `handleRequest()` method is called on the class that has control. Control starts with [`Director`](api:SilverStripe\Control\Director).
```

Reply:
Updated